src/components/Stepper/src/Stepper.vue

```diff
@@ -124,22 +155,57 @@ export default {
 				return;
 			}
 			const incrementBy = 1;
-			/**
-			 * updated stepper value
-			 * @property {number}
-			 */
-			this.$emit('stepper:update', this.value + incrementBy);
+			this.emitUpdate(this.value + incrementBy);
 		},
 		decrement() {
 			if (!Number.isNaN(this.minVal) && this.value <= this.minVal) {
 				return;
 			}
 			const decrementBy = 1;
+			this.emitUpdate(this.value - decrementBy);
+		},
+
+		triggerManualInput() {
+			this.manualValue = this.value;
+			this.isSettingManualValue = true;
+			this.$nextTick(() => {
+				this.$refs.manualInput.focus();
+				this.$refs.manualInput.select();
+			});
+		},
+
+		commitManualValue(event) {
+			event?.preventDefault?.();
+			event?.stopPropagation?.();
```

Comment:
I haven't seen optional chaining like this on event handling before. Events should always exist and have the same properties, so is this necessary?

Reply:
I was being overly cautious here, I'll update